src/compiler/cell_manager.rs

```diff
@@ -84,7 +77,11 @@ impl Placement {
     }
 
     pub fn first_step_height(&self) -> u32 {
-        self.steps.values().next().unwrap().height
+        if let Some(step) = self.steps.values().next() {
+            step.height
+        } else {
+            self.base_height
+        }
     }
```

Comment:
Is this for circuits with no step, e.g. lookup table only circuits? Trying to envision how base_height is useful when there's no step.

Reply:
Yes, circuits with no steps like the mimc7 constant circuit. It will also help for circuits with steps of different height.